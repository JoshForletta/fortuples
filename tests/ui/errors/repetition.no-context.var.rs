use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    impl Container<#Tuple> {
        #myvar
    }
}

fn main() {}
