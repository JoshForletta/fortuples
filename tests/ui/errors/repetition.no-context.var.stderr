error: Attempting to expand a tuple variable without the repetition context.
       Try rewrite this like the following: `#(#myvar),*`
 --> tests/ui/errors/repetition.no-context.var.rs:7:10
  |
7 |         #myvar
  |          ^^^^^
